x/stakeibc/types/params.go

```diff
@@ -16,37 +16,39 @@ var (
 	DefaultRewardsInterval        uint64 = 1
 	DefaultRedemptionRateInterval uint64 = 1
 	// you apparently cannot safely encode floats, so we make commission / 100
-	DefaultStrideCommission           uint64 = 10
-	DefaultICATimeoutNanos            uint64 = 600000000000
-	DefaultBufferSize                 uint64 = 5             // 1/5=20% of the epoch
-	DefaultIbcTimeoutBlocks           uint64 = 300           // 300 blocks ~= 30 minutes
-	DefaultFeeTransferTimeoutNanos    uint64 = 1800000000000 // 30 minutes
-	DefaultMinRedemptionRateThreshold uint64 = 90            // divide by 100, so 90 = 0.9
-	DefaultMaxRedemptionRateThreshold uint64 = 150           // divide by 100, so 150 = 1.5
-	DefaultMaxStakeICACallsPerEpoch   uint64 = 100
-	DefaultIBCTransferTimeoutNanos    uint64 = 1800000000000 // 30 minutes
-	DefaultSafetyNumValidators        uint64 = 35
-	DefaultSafetyMaxSlashPercent      uint64 = 10
+	DefaultStrideCommission                   uint64 = 10
+	DefaultICATimeoutNanos                    uint64 = 600000000000
+	DefaultBufferSize                         uint64 = 5             // 1/5=20% of the epoch
+	DefaultIbcTimeoutBlocks                   uint64 = 300           // 300 blocks ~= 30 minutes
+	DefaultFeeTransferTimeoutNanos            uint64 = 1800000000000 // 30 minutes
+	DefaultMinRedemptionRateThreshold         uint64 = 90            // divide by 100, so 90 = 0.9
+	DefaultMaxRedemptionRateThreshold         uint64 = 150           // divide by 100, so 150 = 1.5
+	DefaultMaxStakeICACallsPerEpoch           uint64 = 100
+	DefaultIBCTransferTimeoutNanos            uint64 = 1800000000000 // 30 minutes
+	DefaultSafetyNumValidators                uint64 = 35
+	DefaultSafetyMaxSlashPercent              uint64 = 10
+	DefaultValidatorExchangeRateQueryInterval uint64 = 100_000_000_000 // denominated in number of native tokens
```

Comment:
I think we're gonna need one of these per host zone. With evmos that has 1e18 denoms this QueryInterval would trigger an ICQ every LSMliquidstake

Reply:
shucks I forgot about evmos, but subconsciously knew we were going to have to do this per host zone. Tbh, I wonder if it's time to go back to the drawing board with the slash query - seems like it's introduced a lot of design complexity for a rare edge case